src/CsvTextEditor/Constants.cs

```diff
@@ -37,6 +37,11 @@ public static class AutomaticUpdates
     public static class Configuration
     {
         public const string CustomEditor = "Settings.Application.Editor.CustomEditor";
+        public const string TimedAutoSave = "Settings.Application.Editor.AutoSaveEditor";
+
+        public static double AutoSaveTimeInSeconds { get { return 60.0; } }
+
+        public const bool AutoSaveDefault = false;
```

Comment:
This should be the same as the setting, thus AutoSaveEditorDefaultValue and the other one should be called AutoSaveEditor

Reply:
Hi Geert, I've standardised everywhere on "AutoSaveEditor" but the GetRoamingValue still only ever seems to return false. I'm sure it's something really simply I've missed, can you see anything obvious that I've missed? 

Cheers

Garry

Reply:
I just pushed a commit named "configuration changes". Just tested and that seems to work fine.

Reply:
OK, will check that out. Cheers.